profiles/README.md

```diff
@@ -10,13 +10,44 @@ Specifically, we include:
 2. Processed data for each Cell Painting plate (for several "data levels")
 3. Instructions on how to reproduce the profiling pipeline
 
-## Workflow
+## Workflow 
 
-![Cytominer workflow](media/cytominer_workflow.png)
+![Cytominer workflow](media/cytominer_workflow_new.png)
+
+## Pycytominer
+[Pycytominer](https://github.com/cytomining/pycytominer) is a code base built by @gwaygenomics and @niranjchandrasekaran. 
+It allows easy processing CellProfiler data and contains all functions that were used to create the data in this repository. Below, we describe the different steps of the pipeline. Please check the pycytominer repo for more details. 
+
+Part of the pipeline, from Level 3 to Level 4b, can be found in the [profile_cells](https://github.com/broadinstitute/lincs-cell-painting/blob/master/profiles/profile_cells.py) script and the final aggregation to the consensus data is found in this [notebook](https://github.com/broadinstitute/lincs-cell-painting/blob/master/consensus/build-consensus-signatures.ipynb).  
 
 Note here that we do not include the intermediate step of generating `.sqlite` files per plate using a tool called [cytominer-database](https://github.com/cytomining/cytominer-database).
 This repository and workflow begins after we applied cytominer-database.
 
+
+### Aggregation
+The [aggregation method](https://github.com/cytomining/pycytominer/blob/master/pycytominer/aggregate.py) is used twice in the workflow. Firstly, the median of all cells within a well is aggregated to one profiler per well. The aggregation method doesn't persist the metadata which is why this step is followed by an annotation step to add the MOA data and others. 
```

Comment:
I like the idea of thinking about creating the consensus profiles as another aggregation step. But given that you have a separate section for consensus below perhaps you could talk about aggregating 4b to consensus profiles in that section and not here.

Reply:
@michaelbornholdt - can you address this?

Reply:
I have, there are two different sections now that explain aggregation and consensus.